thoth/storages/graph/postgres.py

```diff
@@ -573,6 +573,35 @@ def python_package_exists(self, package_name: str) -> bool:
                 > 0
             )
 
+    def solved_software_environment_exists(self, os_name: str, os_version: str, python_version: str) -> bool:
+        """Check if there are any solved packages for the given software environment."""
+        with self._session_scope() as session:
+            result = session.query(
+                session.query(PythonPackageVersion)
+                .filter(
+                    PythonPackageVersion.os_name == os_name,
+                    PythonPackageVersion.os_version == os_version,
+                    PythonPackageVersion.python_version == python_version,
+                )
+                .exists()
+            ).scalar()
+
+            return result
+
+    def get_solved_software_environment_configurations(self) -> List[Dict[str, str]]:
```

Comment:
I don't think it follow the query convention. `get_<objects>_all()` where objects are entities from the schema table (e.g. PythonPackageVersion). In this case, we are taking property of PPV, not an entity from the schema, so it would be confusing maybe. WDYT? What about `get_solved_python_package_versions_software_environment_all`? (even if solved can be implicit because PPV are solved by definition)

Reply:
Awesome! Thanks for the review. Adjusted in 68c4482